src/utils/sign.ts

```diff
@@ -20,6 +21,7 @@ export interface AssuredSignInfo {
   arguments: Buffer;
   decodedArguments?: JsonValue;
   manual: boolean;
+  preAprove: boolean;
```

Comment:
I guess it should be

```suggestion
  preApprove: boolean;
```